pom.xml

```diff
@@ -90,18 +90,17 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>structs</artifactId>
-            <version>1.4</version>
+            <version>1.5</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps</artifactId>
-            <version>2.10</version>
-            <scope>test</scope>
+            <version>2.18-SNAPSHOT</version>
```

Comment:
:-1:, do not want this dep here. You can put it in Pipeline Utility Steps, wherever, but I want this plugin kept free of heavy deps.